README.md

```diff
@@ -60,12 +60,13 @@ Then, just start the LemMinX Language Server as documented on the LemMinX docume
 
 ### Eclipse IDE
 
-This extension is used from the "Generic Text Editor" when Eclipse m2e and [Wild Web Developer](https://github.com/eclipse/wildwebdeveloper) are installed.
```

Comment:
@mickaelistria does Wild Web Developer need to be explicitly installed (when installing m2e)? I assumed Wild Web Developer.XML is automatically installed.

Reply:
No, the necessary parts of Wild Web Developer are fetched as transtive dependencies.

Reply:
Ok so this is good to merge then, thanks!